### PR TITLE
Refactor alert plugins and simplify its usage

### PR DESCRIPTION
**Pull request by kezhenxu94**
This follows up https://github.com/apache/dolphinscheduler/pull/6712 and should be merged after https://github.com/apache/dolphinscheduler/pull/6712

@CalvinKirs @dailidong @lenboo please take a look, this greatly simplifies how to develop a new plugin and how to use the plugins, and running locally.

### CODE REVIEW

**Comment by kezhenxu94** [comment]
@CalvinKirs @dailidong @lenboo this can be reviewed now

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuSender.java [hunk 1]
```diff
@@ -31,23 +31,19 @@
...
-
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class FeiShuSender {
-
-    private static final Logger logger = LoggerFactory.getLogger(FeiShuSender.class);
-
-    private String url;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
```
Comment (davidzollo) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuSender.java]: don't use lombok, thx

Comment (kezhenxu94) [reply dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuSender.java]: Done

**Comment by kezhenxu94** [comment]
Hey @ruanwenjun , wanna take a look at this?

**Review by davidzollo — APPROVED** [review]
+1
great job

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**